enhancements/windows-containers/disconnected-support.md

```diff
@@ -0,0 +1,372 @@
+---
+title: winc-support-in-disconnected-environments
```

Comment:
Should this be reframed as Windows node support for image mirroring?
All the work seems to be around this

Reply:
I was wrestling with this as well but was not 100% sure since we are adding support for specific image mirroring methods only, the minimum required to support disconnected envs. The Non-Goals section outlines this, let me know if you think we should reframe still